apps/webapp/app/presenters/JobPresenter.server.ts

```diff
@@ -0,0 +1,26 @@
+import { User } from "~/models/user.server";
```

Comment:
This looks to be code from another PR, could you remove all this and only leave the changes for the CLI?

Reply:
Okay. I'll tend to it.